tests/main/lxd-mount-units/task.yaml

```diff
@@ -5,7 +5,8 @@ details: |
     are correct for snapfuse.
 
 # only 20.04+, we want lxd images that come with snaps preinstalled.
-systems: [ubuntu-2*]
+# TODO: enable for ubuntu-21.04-64 when the ñxd image is available
+systems: [ubuntu-2*, -ubuntu-21.04-64]
```

Comment:
Unfortunately spread does not support mixing systems and negation of systems :/

Reply:
fixed
Thanks